TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -10,6 +10,20 @@
 
 namespace TPP.Core.Commands.Definitions
 {
+    /// <summary>
+    /// Helper class to store all relevant Information for a single Region
+    /// </summary>
+    public class RegionInformation
+    {
+        public Generation Generation { get; }
+        public int TotalRegionCount { get; }
+        public RegionInformation(Generation generation, int totalRegionCount)
+        {
+            Generation = generation;
+            TotalRegionCount = totalRegionCount;
+        }
+    }
```

Comment:
You may simply use a record for this:

```suggestion
    public record RegionInformation(Generation Generation, int TotalRegionCount);
```